spec/integration/layout_periphery_boundingbox_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'when configuring the header and footer background colors' do
+
+  class PeripheryExtensionTest < ::Asciidoctor::Pdf::Converter
+
+    def build_pdf_options doc, theme
```

Comment:
It's now possible to pass these overrides to the `to_pdf` helper using the `:theme_overrides` option.